test-resources/bin/util

```diff
@@ -26,13 +26,6 @@ setup_test_environment () {
   test-resources/bin/render-templates.sh
 }
 
-install_cert_manager_if_necessary () {
```

Comment:
issue: This is still called in line 25. The call needs to be removed as well.

Reply:
Are you sure? The code compiles on my end.

Reply:
It is called in a bash script that you have not used so far I think. Nevermind, I take care of it.

Reply:
fixed